engine/baml-lib/jsonish/src/tests/test_maps.rs

```diff
@@ -0,0 +1,45 @@
+use super::*;
```

Comment:
@sxlijin some more tests would be nice

map with new lines, for keys,

partial maps
nested partial maps
maps with ints as keys to ensure they cast to strings
maps with arrays as key to ensure they cast to strings
maps in json markup
nested maps with new lines
union of map vs class (would the map correctly let the class have it?)